src/Logger.ts

```diff
@@ -0,0 +1,26 @@
+import consolaGlobalInstance, { LogLevel } from 'consola'
+
+export default consolaGlobalInstance
+consolaGlobalInstance.level = 4
```

Comment:
Can we use one of [consola's constants](https://github.com/nuxt-contrib/consola/blob/master/src/types.js) so it's a bit clearer than this? Or even just a comment would help